docs/resources/identity_agency.md

```diff
@@ -9,6 +9,7 @@ This is an alternative to `huaweicloud_iam_agency_v3`
 
 ## Example Usage
 
+### Delegate the operation authority of the resources in the account to other Huaweicloud accounts
```

Comment:
Delegate another HUAWEI CLOUD account to perform operations on your resources.

Reply:
thanks for suggestion, it's updated at document.